### PR TITLE
changes to 02 do file

### PR DESCRIPTION
**Pull request by krubio25**

### CODE REVIEW

File: dofile/02_nolearning_130.do [hunk 1]
```diff
@@ -45,31 +52,17 @@ use analysis,clear
...
-	*di ((890/4.3)/${exchangerate})*12*2.85 = 1090
-
 
 *********** Cumulative Rug Production ***********
 ** code for Magdy's cumulative production
 ** send into excel
-use "${RAfolder}/Data/Rug Survey/Rug Surveys.dta", clear
+use "$data/Rug Surveys.dta", clear
 
 	foreach var in  order_date order_due_date order_submitted_date {
```
Comment (krubio25) [inline dofile/02_nolearning_130.do]: These variables (order_date, order_due_date, order_submitted date) are not found in the Rug Surveys.dta file. However, there are variables that are rs_order_date, rs_order_due_date, rs_order_submitted date. Is this line referring to those instead?

Comment (krubio25) [inline dofile/02_nolearning_130.do]: @adammosman _order_year_ is not found in the same data file. Neither an _rs_order_year_

**Comment by adammosman** [comment]
It looks like the file was an old one. I've replaced it with the correct on
in dropbox.

File: dofile/02_nolearning_130.do [hunk 2]
```diff
@@ -150,11 +143,11 @@ use analysis,clear
 	drop if id==328
 
 	*# of Firms & # in Treatment
-	tabout etype treatment if inlist(round,100,200) using "${tables}num_treatment.xls", replace
-	tabout etype takeup if round==203 using "${tables}num_treatment.xls", append
+	tabout etype treatment if inlist(round,100,200) using "$tables/num_treatment.xls", replace
+	tabout etype takeup if round==203 using "$tables/num_treatment.xls", append
```
Comment (krubio25) [inline dofile/02_nolearning_130.do]: @adammosman gets an error on this one with the following error:
"Some of your variables consist entirely of missing values"
"==1 invalid name"

File: dofile/02_nolearning_130.do [hunk 3]
```diff
@@ -1,25 +1,32 @@
...
-	*I think this gives us the most accurate cross-sectional look at these costs
-	*run only on control firms (note that baseline data for these variables not trustworthy)
-	noi tabstat weft_cost warp_cost gross_m2_price weaving_time_artisan if round==103 & inlist(product_type,1,2,3,5) & treatment==0, by(product_type) s(mean sd) not c(v)
+***Create Samples****
+	global dubs_joint     = "inlist(strata,1,2,3,4,21,22,23,24) & !inlist(round,100,200)"
+	global dubs_sampleJ   = "inlist(strata,1,2,3,4,21,22,23,24) & !inlist(round,100,200)"
+	global dubs_sample2   = "inlist(strata,21,22,23,24) & !inlist(round,100,200)"
+	global dubs_sample1   = "inlist(strata,1,2,3,4) & !inlist(round,100,200)"
+	global dubs           = "inlist(strata,1,2,3,4,21,22,23,24)"
+	global sample2noround = "inlist(strata,21,22,23,24)"
```
Comment (krubio25) [inline dofile/02_nolearning_130.do]: @adammosman  Should i also carry over the following codes from the 01 do file?
	global exchangerate = 6.31
	gen control_mean=1

Comment (adammosman) [inline dofile/02_nolearning_130.do]: Not sure. Might be worth adding for now. I'm not sure if we actually use it anywhere.

File: dofile/02_nolearning_130.do (hunk 2 above)
Comment (krubio25) [reply dofile/02_nolearning_130.do]: @adammosman  so its using tabout if round==203 but there is no values of 203 in the _round_ variable in the analysis.dta file
_

**Comment by adammosman** [comment]
Weird. Might be slightly old code, not sure why though. Use round 210
instead of 203

File: dofile/02_nolearning_130.do [hunk 4]
```diff
@@ -166,7 +164,7 @@ use analysis,clear
 	reg produced if inlist(round,203) & takeup==1 & inlist(etype,1,5)
```
Comment (krubio25) [inline dofile/02_nolearning_130.do]: @adammosman getting an error "no observations" after the loop. What regression estimates is this loop storing? What is _produced_ being regressed by?

**Comment by adammosman** [comment]
This was done just to get us the mean for "*produced" *conditional on
takeup and the type of rug. You still get no observations if you change to
round 210?

Now that I think of it it may be that the other dataset that holds the
"produced" variable is what holds round 203. You might need to change the
round in that dataset before merging with analysis

File: dofile/02_nolearning_130.do (hunk 4 above)
Comment (krubio25) [reply dofile/02_nolearning_130.do]: @adammosman yes, i still get no observations if I change to round 210.

**Comment by adammosman** [comment]
Seems like I had included an older takeup.dta file. Sorry. I've replaced it
with the correct one now which should work with round 203

**Pull request merged**